app/sprinkles/admin/templates/pages/dashboard.html.twig

```diff
@@ -245,9 +245,13 @@
                   <img class="img-circle" src="{{assets.url('assets://local/core/images/cupcake.png')}}" alt="User Avatar">
                 </div>
                 <div class="box-footer">
-                    <h4>Welcome to {{site.title}}!</h4>
+                    <h4>
+                        {{translate("WELCOME_TO", {
+                            'title': site.title
+                        })}}
+                    </h4>
                     <p>
-                        It doesn't look like any features have been set up for this account...yet.  Maybe they haven't been implemented yet, or maybe someone forgot to give you access.  Either way, we're glad to have you aboard!
+                        {{translate("NO_FEATURES_YET")}}
```

Comment:
Not sure this need translating, at least not in core sprinkles where the key will stay forever. This sentence will be changed by devs in 95% of cases maybe...